tasks/views.py

```diff
@@ -60,7 +65,9 @@ def dispatch(self, *args, **kwargs):
     def get(self, request):
         all_tasks = {}
         if request.user.is_authenticated:
-            all_tasks = request.user.task_set.all()
+
+            user = User.objects.prefetch_related('task_set').get(email=request.user.email)
```

Comment:
I didn't work when I tried. Upon looking up I found that prefetch_related works on a queryset object. get() returns a single model, not a queryset. However, we can change the order if we use `filter()`  instead of `get()`